models/org.go

```diff
@@ -476,7 +476,8 @@ func GetOrgsCanCreateRepoByUserID(userID int64) ([]*User, error) {
 		Join("INNER", "`team`", "`team`.id = `team_user`.team_id").
 		Where(builder.Eq{"`team_user`.uid": userID}).
 		And(builder.Eq{"`team`.authorize": AccessModeOwner}.Or(builder.Eq{"`team`.can_create_org_repo": true})))).
-		Desc("`user`.updated_unix").Find(&orgs)
+		Asc("`user`.name").
```

Comment:
Would lower name be better? Or should we have the capitalised organizations before the non-capitalised ones?

```suggestion
		Asc("`user`.lower_name").
```

Reply:
Database will sort using natural language rules so it does not matter much